mesonbuild/modules/pkgconfig.py

```diff
@@ -441,7 +441,7 @@ def _generate_pkgconfig_file(self, state: ModuleState, deps: DependenciesHelper,
                                  pkgroot: T.Optional[str] = None) -> None:
         coredata = state.environment.get_coredata()
         referenced_vars = set()
-        optnames = [x.name for x in BUILTIN_DIR_OPTIONS.keys()]
+        optnames = [x.name for x in BUILTIN_DIR_OPTIONS]
```

Comment:
python also supports this:
```suggestion
        referenced_vars = set(); optnames = [x.name for x in BUILTIN_DIR_OPTIONS]
```

It, too, is fewer lines. It, too, is widely considered less readable.

Reply:
PEP8